api/envoy/api/v2/route/route.proto

```diff
@@ -773,6 +773,14 @@ message RouteAction {
     google.protobuf.BoolValue enabled = 2;
   };
   repeated UpgradeConfig upgrade_configs = 25;
+
+  // Configures :ref:`internal redirect <arch_overview_internal_redirects>` behavior.
+  enum InternalRedirectAction {
+    REJECT = 0;
+    PASS_THROUGH = 1;
```

Comment:
these need to be swapped IMO. Passthrough is default, reject is option 1.

Reply:
+1 for swapping, though per the big convo below I think this is going to get changed anyway.